### DISCUSSION TITLE
Why doesn't Renovate create PRs when it detects new versions of a package?

### DISCUSSION THREAD

**Discussion by nfrankel** [mend/renovate-ce-ee#344] [Q&A] [answered]
I've configured Renovate on a private repo; it does indeed look the dependencies and find newer versions.

However, it doesn't create any PR (see attached log). Can somebody please help?

[renovate.log](https://github.com/mend/renovate-ce-ee/files/12788992/renovate.log)

**Comment by rarkins** [comment, marked as answer]
Renovate has been configured to use "Silent" mode in the hosted app. This gets converted to dryRun=lookup in config, which. means no PRs. You can go to https://developer.mend.io and change the org or repo setting from Silent to Interactive

**Reply by rarkins** [reply]
FYI to yourself or anyone reading it, this is default suggested option when onboarding a new org after installation. The idea is that you can see a preview in the app's UI first if you prefer instead of straight into onboarding PRs

**Comment by nfrankel** [comment]
Thanks a lot!!!
